ui/webui/test/check-language

```diff
@@ -114,5 +114,20 @@ class TestLanguage(anacondalib.VirtInstallMachineCase):
         l.check_selected_locale("de_DE")
         b.wait_attr("html", "dir", "ltr")
 
+    def testLanguageScreenHiding(self):
+        self.restore_file('/run/anaconda/anaconda.conf')
+        b = self.browser
+        m = self.machine
+        i = Installer(b, m)
+
+        m.execute("sed -i 's/type = BOOT_ISO/type = LIVE_OS/g' /run/anaconda/anaconda.conf")
+
+        # Need to wait for storage-devices step because installation-language was hidden
+        i.open(step="storage-devices")
```

Comment:
So you want to test that opening without specifying which step which automatically pick the storage-devices step. So do i.open(step='')

Reply:
First line of this function run well but ```self.wait_current_page(step)``` cant work with empty string. But we need to wait for the page there. Maybe original ```i.open(step="storage-devices")``` is there right because we cant wait at BootIso for this page without clicking on the next button.

Reply:
Since this fakes the media to looks like live and not boot.iso, there is never the next to click next as there is not welcome page shown. 

Please sent the latest version of the code and I can take a look how it fails.

Reply:
Its its too difficult for this type of test update helpers. If i use ```i.open(step="storage-devices")``` it can only end with timeout if its not right.